include/Register.h

```diff
@@ -72,13 +72,7 @@ class EDB_EXPORT Register {
 	T value() const             { return T(value_); }
 
 	// Return the value, zero-extended to address_t to be usable in address calculations
-	edb::address_t valueAsAddress() const {
-		// This function only makes sense for GPRs
-		assert(bitSize_ <= 8 * sizeof(edb::address_t));
-		edb::address_t result(0LL);
-		std::memcpy(&result, &value_, bitSize_ / 8);
-		return result;
-	}
+	edb::address_t valueAsAddress() const;
```

Comment:
This is a bit of a pessimization: the compiler is now unable to inline the (trivial!) function unless link-time optimization does this.

Reply:
Yea, on this one. I kinda tend to prefer to have the implementation in the source files unless it is something like a one-liner. If this ever shows up in a profile as being notably slow, well you can say you told me so ;-).